cumulusci/cli/service.py

```diff
@@ -321,18 +351,30 @@ def get_service_data(service_config, sensitive_attributes) -> list:
     return service_data
 
 
-def get_sensitive_service_attributes(runtime, service_type) -> list:
+def get_service_attributes(
+    runtime: CliRuntime, service_type: str, sensitive: bool = False
```

Comment:
Maybe the parameter should be called only_sensitive_attributes to differentiate from the equally plausible include_sensitive_attributes?